code/game/objects/structures/watercloset.dm

```diff
@@ -420,7 +420,7 @@
 	name = "sink"
 	icon = 'icons/obj/watercloset.dmi'
 	icon_state = "sink"
-	desc = "A HV-11 sink. Used for washing the hands and face."
+	desc = "A HV-11 sink used for washing hands and face."
```

Comment:
Suggest also changing the first "A" to "An HV-11..." due to the quirks of when to use A vs. An and the phonetics of saying the letters 'HV'.

Reply:
This is fair, yes!